src/main/java/core/basesyntax/ReturnOperation.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Map;
+
+public class ReturnOperation implements OperationHandler {
```

Comment:
The class implementation should not begin with an empty line. According to the checklist, you should remove all redundant empty lines.